web/sites/default/config/webform.webform.contact_ppd.yml

```diff
@@ -0,0 +1,375 @@
+          value: 'Yes'
+    '#location_address__title': 'Property address'
+    '#location_address__description': 'Begin typing a Portland street address to see a list of possible matches and select one, or enter a partial address and click Verify.'
+    '#location_address__required': true
+    '#location_full_address__description': 'This is the full address'
+    '#unit_number__description': 'This is the unit number'
+    '#location_city__description': 'This is the city'
+    '#location_state__description': 'This is the state'
+    '#location_zip__description': 'This is the zipcode'
+    '#location_full_address__wrapper_attributes':
+      class:
+        - d-none
+    '#unit_number__wrapper_attributes':
+      class:
+        - d-none
+    '#location_city__wrapper_attributes':
+      class:
+        - d-none
+    '#location_state__wrapper_attributes':
+      class:
+        - d-none
+    '#location_zip__wrapper_attributes':
+      class:
+        - d-none
+    '#require_portland_city_limits': 1
+    '#lookup_taxlot': 1
+  computed_verified_address:
+    '#type': webform_computed_twig
+    '#title': 'Full Address'
+    '#title_display': none
+    '#states':
+      visible:
+        ':input[name="location_address[location_verification_status]"]':
+          value: Verified
+    '#display_on': form
+    '#template': '<strong>Full verified address:</strong> {{ data.location_address.location_full_address }}'
+    '#ajax': true
+    '#lookup_taxlot': 1
+  request_number_type:
+    '#type': radios
+    '#title': 'Do you have a permit, IVR, or case number?'
+    '#options':
+      'I have a permit number': 'I have a permit number'
+      'I have an IVR number': 'I have an IVR number'
+      'I have a case number': 'I have a case number'
+      "I don't know my number": "I don't know my number"
+    '#required': true
+  request_permit_number:
+    '#type': textfield
+    '#title': "What's the number?"
+    '#required': true
+    '#states':
+      visible:
+        - ':input[name="request_number_type"]':
+            value: 'I have a permit number'
+        - or
+        - ':input[name="request_number_type"]':
+            value: 'I have an IVR number'
+        - or
+        - ':input[name="request_number_type"]':
+            value: 'I have a case number'
+  request:
+    '#type': textarea
+    '#title': "What's your question or request?"
+    '#required': true
+  file_upload:
+    '#type': managed_file
+    '#title': 'Upload related files'
+    '#multiple': 5
+    '#description': '<p>Add files like permits, documents, or drawings to help us assist you.</p>'
+    '#description_display': before
+    '#file_extensions': 'jpg pdf'
+    '#sanitize': true
+  computed_address_link:
+    '#type': webform_computed_twig
+    '#title': 'Computed Address Link'
+    '#display_on': none
+    '#mode': text
+    '#template': "https://www.portlandmaps.com/search/?query={{ data.location_address.location_address|replace({' ': '%20'}) }}"
```

Comment:
|[url_encode](https://twig.symfony.com/doc/3.x/filters/url_encode.html#url-encode) can be used instead to cover all special characters

Reply:
Resolved